selendroid-standalone/src/test/java/io/selendroid/standalone/android/impl/DefaultHardwareDeviceTests.java

```diff
@@ -61,6 +62,7 @@ private void cleanUpDevice(AndroidDevice emulator) throws Exception {
     Assert.assertFalse(installedAPKs.contains(AUT_PACKAGE));
   }
 
+  @Ignore("Fail. Requires an active emulator to install the selendroid apk.")
```

Comment:
the comment in this test file says `Works only if ... one emulator is already running`